definitions/npm/commander_v2.x.x/flow_>=0.28.x/commander_v2.x.x.js

```diff
@@ -0,0 +1,276 @@
+declare class $npm$commander$Command extends events$EventEmitter {
```

Comment:
Can we just move this inside the `declare module` body? Best to avoid globals unless they're strictly needed.

(It's always possible to move it *out* of the declare module body later if some other libdef needs to reference the type before we've gotten around to implementing imports for `declare module`s)